modules/user-access/pages/user-management.adoc

```diff
@@ -9,10 +9,18 @@ To see user management tasks under the xref:access-control-model.adoc#_access_co
 
 You can run the `CREATE USER` command to create a user.
 
-The username cannot contain the following characters: `\` ,` ` `(` , `)`, `[`, `]`, `:`, `<`, `>`, `;`, `,`, `@`, `\r`, `\n`, `\f`, `\t`, `\\`, `\0`, `\b`.
-It also cannot start with a dot `.` or have multiple dots in a sequence.
+*Usernames* can contain any Unicode characters
+or any valid email *except* the following:
 
-You can use non-ascii characters, such as Chinese and Kanji characters.
```

Comment:
@JosephNewmanTG I'm not seeing any change since the lengthy discussion in Jira.
The last time I looked, the engineer hadn't told us what he is actually checking. E.g.
* What is the actual rule about backquotes?
* What is the actual rule about colon?
* The final version needs to be grammatically = semantically correct. Go for complete sentences, with an explicitly stated subject for each sentence, e.g., "A <subject> may not <the prohibition>."